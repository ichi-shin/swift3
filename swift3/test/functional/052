#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-4-6_2

. ./common

S3USER=tester

_s3_delete /bucket | _filter_curl xml
