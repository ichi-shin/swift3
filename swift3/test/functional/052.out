QA output created by 052
> s3curl --id tester -- -X DELETE http://SWIFT_HOST/bucket... 404
<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchBucket</Code>
  <Message>The specified bucket does not exist.</Message>
  <RequestId>TXID</RequestId>
  <BucketName>bucket</BucketName>
</Error>
